lib/src/vector_math/matrix2.dart

```diff
@@ -78,6 +78,18 @@ class Matrix2 {
   factory Matrix2.rotation(double radians) =>
       Matrix2.zero()..setRotation(radians);
 
+  /// Sets the value for row 0 and column 0.
+  set r0c0(double value) => _m2storage[0] = value;
```

Comment:
Does this provide any benefits over `setEntry(x, y, value)`?
Won't this just be another way to do the same thing?

Reply:
Implementation of `setEntry`:
```dart
  void setEntry(int row, int col, double v) {
    assert((row >= 0) && (row < dimension));
    assert((col >= 0) && (col < dimension));

    _m2storage[index(row, col)] = v;
  }

  int index(int row, int col) => (col * 2) + row;
```

Implementation of `r0c0`:
```dart
set r0c0(double value) => _m2storage[0] = value;
```

I believe `r0c0` is less verbose than `setEntry` and it will be more performant than `setEntry`.

Reply:
Good suggestion @spydon! 

Here is what I got:
![Screenshot 2022-06-08 at 19 03 59](https://user-images.githubusercontent.com/44524995/172686264-0b8bbaf3-8427-46b6-8e83-eb7b397ea97e.png)

So `r0c0 = 1` is faster than `setEntry(0, 0, 1)`.

Here is the code I used:

```dart
final stopwatch = Stopwatch()..start();
for (var i = 0; i < 100; i++) {
  Matrix2.zero().setEntry(0, 0, 1);
}
stopwatch.stop();
print(stopwatch.elapsedMicroseconds);
```

```dart
 final stopwatch = Stopwatch()..start();
for (var i = 0; i < 100; i++) {
  Matrix2.zero().r0c0 = 1;
}
stopwatch.stop();
print(stopwatch.elapsedMicroseconds);
```

Results varied from time to time but `r0c0` was always faster.

I'm running using MacBook Pro M1 (macOS 12.2 21D49 darwin-arm).

Hope this helps.

Reply:
Thanks for the benchmark, have you moved in the index calculation to be inside of `setEntry` so that it doesn't have to do that unnecessary extra method call?

Reply:
No, the benchmark is with the code in this PR. 

I'm intrigued to know the difference without the index call. I will look into it tonight or tomorrow.